src/main/java/minicraft/level/LevelGen.java

```diff
@@ -794,10 +834,7 @@ public static void main(String[] args) {
 			}
 			img.setRGB(0, 0, w, h, pixels, 0, w);
 			JOptionPane.showMessageDialog(null, null, "Another Map", JOptionPane.PLAIN_MESSAGE, new ImageIcon(img.getScaledInstance(w * 4, h * 4, Image.SCALE_AREA_AVERAGING)));
-			if (LevelGen.worldSeed == 0x100)
-				LevelGen.worldSeed = 0xAAFF20;
-			else
-				LevelGen.worldSeed = 0x100;
+			LevelGen.worldSeed++;
```

Comment:
Are you sure this doesn't break anything?

Reply:
Why?

Reply:
This is just a seed.

Reply:
And I have slightly updated a bit.